playbooks/demo_flrtvc_dnf.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: "FLRTVC on AIX playbook"
+  hosts: all
+  user: root
+  gather_facts: no
+  vars:
+    apar: sec
+    path: /usr/sys/inst.images
+    check: no
+    download_only: yes
+    extend: yes
+    ansible_python_interpreter: /opt/freeware/bin/python3
+    PATH: /opt/freeware/bin
+  collections:
+  - ibm.power_aix
+
+  tasks:
+  - name: Check for wget command
+    dnf:
+      name: "wget"
```

Comment:
you already have a bootstrap feature to install wget. i think you can use the import role here for bootstrap role